suite-native/app/ios/Podfile

```diff
@@ -21,7 +21,7 @@ target 'TrezorSuite' do
     #
     # Note that if you have use_frameworks! enabled, Flipper will not work and
     # you should disable the next line.
-    :flipper_configuration => FlipperConfiguration.enabled,
+    :flipper_configuration => FlipperConfiguration.enabled(["Debug", "Develop"], { 'Flipper' => '0.171.1' }),
```

Comment:
I would suggest to put only Debug here, we don't need flipper in builds from develop branch

Reply:
This is not branch but build type. That means when you build locally through XCode or CLI, it will include Flipper pod.